Sofa/Component/AnimationLoop/src/sofa/component/animationloop/FreeMotionAnimationLoop.cpp

```diff
@@ -403,9 +403,12 @@ void FreeMotionAnimationLoop::computeFreeMotion(const sofa::core::ExecParams* pa
     }
 }
 
-int FreeMotionAnimationLoopClass = core::RegisterObject(R"(
-The animation loop to use with constraints.
-You must add this loop at the beginning of the scene if you are using constraints.")")
-                                   .add< FreeMotionAnimationLoop >();
+void registerFreeMotionAnimationLoop(sofa::core::ObjectFactory* factory)
+{
+    factory->registerObjects(core::ObjectRegistrationData(R"(
+        The animation loop to use with constraints.
+        You must add this loop at the beginning of the scene if you are using constraints.")")
```

Comment:
To be checked, but I think the indentation you have in a raw string are taken into account in the string.

Reply:
you are right, it is taken into account